### ISSUE TITLE
Two PRs for org.freedesktop.Sdk.Extension.openjdk21 are failing due to "couldn't find remote ref" error

### ISSUE THREAD

**Issue by Zishan-Rahman** [flathub/flathub#5601]
The PRs:
1. https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/68
2. https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/69

Both of their test builds fail at the `checkout manifest` step due to the same error, and I'm struggling to find out why these errors are happening and how to fix them (I've tried building test builds 3 times for each PR):
```
fatal: couldn't find remote ref refs/pull/68/head
```
```
fatal: couldn't find remote ref refs/pull/69/head
```

Any advice and/or guidance on this matter is very much appreciated.

**Comment by Zishan-Rahman** [comment]
@hfiguiere (sorry for the ping) I've taken a note of [what you said in](https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/69#issuecomment-2351816547) https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/69 about that PR not being necessary, but what could we do about https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/68 failing (for the same reason as the other PR)? Because if _that_ PR isn't necessary now, it soon will be.

**Comment by bbhtt** [comment]
These seems like an issue on Github's end.

**Comment by bbhtt** [comment]
Seems to be working https://github.com/flathub/org.freedesktop.Sdk.Extension.openjdk21/pull/70

**Comment by bbhtt** [comment]
Somehow Github was thinking the PR was already merged or something. If I closed it `refs/pull/68/head` appeared.
